reversers/database-synchroniser/features/liquibase.feature/feature.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<feature
+      id="liquibase.feature"
+      label="Liquibase Feature"
+      version="4.0.0.qualifier"
+      provider-name="Liquibase"
+      plugin="liquibase">
+
+   <description url="https://www.liquibase.org/">
+      Feature that wrap liquibase into a Eclipse bundle
+   </description>
+
+   <copyright url="https://www.liquibase.org/">
+      https://www.liquibase.org/
+   </copyright>
+
+   <license url="https://www.apache.org/licenses/LICENSE-2.0">
```

Comment:
I'm not sure if we need to reference the EPL licence as well. I'm waiting for an answer from Mélanie on this point.

Reply:
We don't need the Apache license. Can you please do as it is done in the other plugins ?
Thanks :)